package.json

```diff
@@ -56,14 +56,18 @@
     "lint": "eslint src test",
     "lint:fix": "eslint --fix src test",
     "clean:node": "rm -rf index.js lib",
+    "clean:node.win": "RMDIR /S /Q lib && ERASE /F /Q index.js",
     "clean:browser": "rm -rf validator*.js",
+    "clean:browser.win": "ERASE /F /Q validator*.js",
     "clean": "npm run clean:node && npm run clean:browser",
+    "clean.win": "npm run clean:node.win && npm run clean:browser.win",
     "minify": "uglifyjs validator.js -o validator.min.js  --compress --mangle --comments /Copyright/",
     "build:browser": "node --require @babel/register build-browser && npm run minify",
     "build:node": "babel src -d .",
     "build": "npm run build:browser && npm run build:node",
     "pretest": "npm run lint && npm run build",
-    "test": "nyc mocha --require @babel/register --reporter dot"
+    "test": "nyc mocha --require @babel/register --reporter dot",
+    "buildtest": "npm run build && npm run lint:fix && npm run test"
```

Comment:
You don't need to add a buildtest command since ```pretest``` will be called automatically by ```npm test```